Added KafkaStorageDriver class

AnastasiaShemyakinskaya opened the pull request.

AnastasiaShemyakinskaya committed:
Added KafkaStorageDriver class

akurilov commented on src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java:
```diff
@@ -0,0 +1,132 @@
+public class KafkaStorageDriver<I extends Item, O extends Operation<I>>
+    extends CoopStorageDriverBase<I, O> {
+
+  protected KafkaStorageDriver(
```
let the constructor be public

AnastasiaShemyakinskaya replied:
Fixed

akurilov commented on src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java:
```diff
@@ -0,0 +1,132 @@
+
+  @Override
+  protected final int submit(final List<O> ops) throws IllegalStateException {
+    final int opsCount = ops.size();
```
it may be "final var"

AnastasiaShemyakinskaya replied:
Fixed

AnastasiaShemyakinskaya committed:
Fixed constructor and added var
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Anastasia Shemyakinskaya shem98a@gmail.com
